### DISCUSSION TITLE
How to make it so it only shows the line number of the current line I am on

### DISCUSSION THREAD

**Discussion by nikivdev** [zed-industries/zed#36942] [Feature Requests]
### Check for existing issues

- [X] Completed

### Describe the feature

In VSCode I had this:

<img width="879" alt="image" src="https://github.com/zed-industries/community/assets/6391776/91b4ecac-2940-49e8-ad9c-72ceaf62d640">

It showed the line number of current active line, that's it.

I only see this in settings:

<img width="390" alt="image" src="https://github.com/zed-industries/community/assets/6391776/0fdd40cd-3b1d-42de-bd89-fc7d55c26054">

So I don't see how to make above work. I don't even see a setting to turn off line numbers.

**Comment by nikivdev** [comment]
The way it was achieved in VSCode is via this:


```
  "workbench.colorCustomizations": {
    "[GitHub Light]": {
      "editorLineNumber.foreground": "#ffffff",
      "editor.lineHighlightBackground": "#ffffff",
      "editorInlayHint.typeBackground": "#f1f9fd",
      "titleBar.activeBackground": "#ffffff",
      "titleBar.activeForeground": "#0d1116"
    },
    "[GitHub Dark Default]": {
      "editorLineNumber.foreground": "#0c1118",
      "editor.lineHighlightBackground": "#0c1118",
      "editorInlayHint.typeBackground": "#0d1116"
    }
  },
  ```
  
  So I color over the non active line number with the theme color. That way it shows the active line number only.
  
  Can I do same in Zed? 🙏 

I like to keep my UI as minimal as possible. Thank you.

**Comment by JosephTLyons** [comment]
Hey @nikitavoloboev, we don't have a way to achieve this in Zed at the moment.
